ansible/roles/ssc/templates/ssc-deploy.yml.j2

```diff
@@ -0,0 +1,131 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ssc-data
+  namespace: {{ ssc_namespace }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: {{ ssc.volume_size }}
+{% if ssc_storage_class %}
+  storageClassName: {{ ssc_storage_class }}
+{% endif %}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ssc
+  namespace: {{ ssc_namespace }}
+  labels:
+    app: ssc
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ssc
+  template:
+    metadata:
+      labels:
+        app: ssc
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "{{ ssc_metrics_port }}"
+    spec:
+      initContainers:
+        - name: init-data
+          image: busybox
+          command: ['sh', '-c', 'if [ ! -f /data/priv_validator_state.json ]; then echo {"height": "0", "round": 0, "step": 0} > /data/priv_validator_state.json; fi']
+          volumeMounts:
+            - name: ssc-data
+              mountPath: /data
+        - name: init-validator
+          image: {{ ssc_image }}
+          command:
+            - sh
+            - -c
+            - |
+              if [ ! -f /data/config/priv_validator_key.json ]; then
```

Comment:
This should be in a configmap for readability rather than here. Also, why having a start script and also an init script? You only need one.